app/src/main/res/layout/activity_profile.xml

```diff
@@ -10,78 +10,51 @@
     <TextView
         android:id="@+id/profile_text_view"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="150dp"
         android:background="@color/colorPrimary"
-        android:paddingLeft="15dp"
+        android:paddingLeft="30dp"
         android:paddingBottom="5dp"
+        android:paddingTop="75dp"
         android:text="Profile"
         android:textColor="#ffffff"
         android:textSize="40sp"
-        android:textAlignment="center"
         android:textStyle="bold"
         android:layout_centerHorizontal="true"
         tools:layout_editor_absoluteX="0dp"
         tools:layout_editor_absoluteY="0dp" />
 
-    <TextView
-        android:id="@+id/scan_code_text_view"
+    <FrameLayout
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/profile_text_view"
-        android:layout_marginTop="30dp"
-        android:text="@string/ScanQRCodeText"
-        android:textAlignment="center"
-        android:textAllCaps="true"
-        android:textColor="#000000"
-        android:textSize="25sp"
-        android:textStyle="bold" />
+        android:layout_height="wrap_content">
 
-    <ImageView
-        android:id="@+id/qr_code_placeholder_image_view"
-        android:layout_width="250dp"
-        android:layout_height="250dp"
-        android:layout_below="@id/scan_code_text_view"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:layout_centerVertical="true"
-        android:layout_centerHorizontal="true"
-        android:layout_centerInParent="true"
-        android:layout_marginTop="28dp"
-        android:layout_marginEnd="81dp"
-        android:layout_marginRight="81dp"
-        app:srcCompat="@drawable/ic_qr_code" />
+        <!--fragment
```

Comment:
if you don't need this commented out code, could you please remove it from the file (: